Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.cpp

```diff
@@ -533,6 +547,13 @@ namespace O3DE::ProjectManager
             }
 
             m_gemModel->UpdateGemDependencies();
+
+            // if we don't have a project path early out
+            if (m_projectPath == "")
```

Comment:
A better check would be `m_projectPath.isEmpty` as it saves a few cycles.
```suggestion
            if (m_projectPath.isEmpty())
```